### PR TITLE
cleaned up static folder

### PR DESCRIPTION
**Pull request by anahocevar**

### CODE REVIEW

**Comment by ZachGlassman** [comment]
Correct idea here,  however the file you have deleted is still a dependency.  Can you replace https://github.com/thedataincubator/stockpredict/blob/master/stockticker/stockticker.py#L44 with a call to the written `query_quandl` function (don't worry about variable ticker, just use GOOGL) so we will pass tests with the file deletion?

**Comment by ZachGlassman** [comment]
addresses https://github.com/thedataincubator/stockpredict/issues/14

File: stockticker/stockticker.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def create_app(prophet_url, secret_key, quandl_key, bokeh_version): # pylint: di
     def index(): # pylint: disable=W0612
         """main route"""
         # Replace with Quandl API call on user input - may need to edit test
-        df = pd.read_csv('static/GOOGL_data.txt') # pylint: disable=C0103
+        df = query_quandl(GOOGL, quandl_key) # pylint: disable=C0103
```
Comment (ZachGlassman) [inline stockticker/stockticker.py]: GOOGL needs to be a string here

**Review by ZachGlassman — CHANGES_REQUESTED** [review]

File: stockticker/stockticker.py (hunk 1 above)
Comment (anahocevar) [reply stockticker/stockticker.py]: Oops.

Comment (ZachGlassman) [reply stockticker/stockticker.py]: no worries, everything else looks good!

**Comment by ZachGlassman** [comment]
It is failing tests because the Quandl api is not mocked out.  I have just merged a PR which will help resolve this issue.  To pull these changes into your branch, please rebase this branch on the current master.  Here are the steps to do that.

1. `git checkout master`
2. `git pull`
3. `git checkout clean_up_static_folder`
4. `git rebase master`
5. `git push` (if you get an error here, then run `git push -f`).

After this is rebased, then we can alter the test to mock out the Quandl API call (we don't want to make this call from our test).

We can do this by registering a response from the quandl API in both `test_post_error` and `test_post_success`.  To do this, add the following line 
`responses.add(responses.GET, QUANDL_URL, json=QUANDL_DATA, status=200)` 
as the first line of both of those functions!

**Review by ZachGlassman — APPROVED** [review]

**Comment by ZachGlassman** [comment]
LGTM, merge away!

**Pull request merged**